programs/x509/cert_write.c

```diff
@@ -134,6 +135,13 @@ int main(void)
     "    subject_identifier=%%s   default: 1\n"             \
     "                            Possible values: 0, 1\n"   \
     "                            (Considered for v3 only)\n" \
+    "    san=%%s                   default: (none)\n"       \
+    "                            Comma-separated-list of values:\n" \
+    "                             DNS:value\n"            \
+    "                             URI:value\n"            \
+    "                             RFC822:value\n"         \
+    "                             IP:value (Only IPv4 is supported)\n" \
```

Comment:
Why is only IPv4 supported?

Reply:
It should probably be explained in this PR, but it's a compromise step towards the goal.
This will change in a follow-up PR, since https://github.com/Mbed-TLS/mbedtls/pull/7436 (that handles IP parsing with and without platform support) has been merged.